Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -1,49 +1,156 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+"""Tenable.io Demisto integration."""
```

Comment:
```suggestion
```

Reply:
Hi @barchen1 , I verified that the latest version of Tenable_io.py has """Tenable.io Demisto integration.""" on line 4.

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed comment from code